.gitignore

```
@@ -51,4 +51,6 @@ apps/promisetracker/public/data/**
 !apps/promisetracker/public/data/README.md
 
 # Storybook
-storybook-static
\ No newline at end of file
+storybook-static
+
+mongo-keyfile
```

```suggestion
mongo-keyfile

```

Is this file stored somewhere in the vault?

---

[Yes](https://vault.codeforafrica.org/#/vault?search=mong&itemId=7d00fcf3-bc1e-469d-899a-318eea2f3600)